conanfile.py

```diff
@@ -8,7 +8,7 @@
 
 class SISLConan(ConanFile):
     name = "sisl"
-    version = "11.0.6"
+    version = "11.1.1"
```

Comment:
it is backward compatible, right? If so this is correct, no need to bump major version.

Reply:
Yes, it is. This PR adds new APIs and does not touch the existing ones